index.html

```diff
@@ -35,7 +35,119 @@ <h1>I'm David, Glad to see you</h1>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
+                <button type="button">See Project</button>
+            </div>
+            
+        </div>
+        <div class="card">
+            <img class="card-image" src="./img/portfolio3.jpg" alt="webpage image">
+            <h2>Tonic</h2>
+            <div class="tags">
+                <p>CANOPY</p>
+                <img src="./img/Counter.png" alt="bullet">
+                <span>Back End Dev</span>
+                <img src="./img/Counter.png" alt="bullet">
+                <span>2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads. No accounts or 
+                sign-ups required.
+            </p>
+            <ul class="technologies">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-content">
+            <div class="container">
+                <h2>About myself</h2>
+                <p>Hello I'm a software developer! I can help you build a product,  
+                feature or website. Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don't hesitate 
+                to contact me. 
+                </p>
+                <span>LET'S CONNECT</span>
+                <ul class="social-media">
+                    <li><a href="https://twitter.com/DavidEspinoG"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/davidespinog/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
+                    <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
+                    <li><a href="https://github.com/DavidEspinoG"><img src="./img/git.png" alt="github icon"></a></li>
+                    <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
+                </ul>
+                <a><button>Get my resume</button></a>  
+
+                <div class="languages">
+                    <h3>Languages</h3>
+                    <i class="fa-solid fa-chevron-down fa-xl"></i> 
+                </div>
+                <div class="language">
+                    <img class="language-img" src="./img/js.png" alt="JavaScript image">
+                    <p>JavaScript</p>
+                </div>
+                <div class="language">
+                    <img src="./img/html.jpg" alt="JavaScript image">
+                    <p>HTML</p>
+                </div>
+                <div class="language">
+                    <img src="./img/css.jpg" alt="JavaScript image">
+                    <p>CSS</p>
+                </div>
```

Comment:
- Kindly, The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them is a `li` for semantic reasons.👌👌
- Kindly, `HTML` and `CSS` icons are large compared to the Figma design, please minimize its size to stick to the Figma design 👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/212359137-2f9ceb39-b5c6-4224-9cba-4158da1b0fd8.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/212359139-d5312780-2940-4194-8508-7e58342f5c4d.png" width="200">|


😎️

🤝️